### ISSUE TITLE
Error when updating 1.6.0.14 to 1.7.5.2

### ISSUE THREAD

**Issue by jlorquin** [PrestaShop/PrestaShop#14683]
Hi there,

I tried to update from 1.6.0.14 to 1.7.5.2 using 1-click module and received several DB errors.

- The installation .zip was copied into /download (I didn't want to update to 1.7.6.0) and selected manually.
- PHP version is 5.6.40 (7.1.x gave errors when accessing the 1-click upgrade module)
- See screenshot for other options:

![](https://iqit-commerce.com/wp-content/uploads/2017/02/upate.jpg)

Then I get the following errors, all related to the DB step if I'm not mistaken:

`[INTERNAL] /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php line 102 - Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException: The service "prestashop.core.mail_template.theme_folder_catalog" has a dependency on a non-existent parameter "mail_themes_dir". Did you mean this: "themes_dir"? #0 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php(57): Symfony\Component\DependencyInjection\ParameterBag\ParameterBag->get('mail_themes_dir') #1 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php(219): Symfony\Component\DependencyInjection\ParameterBag\EnvPlaceholderParameterBag->get('mail_themes_dir') #2 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php(189): Symfony\Component\DependencyInjection\ParameterBag\ParameterBag->resolveString('%mail_themes_di...', Array) #3 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveParameterPlaceHoldersPass.php(64): Symfony\Component\DependencyInjection\ParameterBag\ParameterBag->resolveValue('%mail_themes_di...') #4 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/AbstractRecursivePass.php(60): Symfony\Component\DependencyInjection\Compiler\ResolveParameterPlaceHoldersPass->processValue('%mail_themes_di...', false) #5 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveParameterPlaceHoldersPass.php(79): Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass->processValue(Array, false) #6 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/AbstractRecursivePass.php(67): Symfony\Component\DependencyInjection\Compiler\ResolveParameterPlaceHoldersPass->processValue(Array) #7 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveParameterPlaceHoldersPass.php(79): Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass->processValue(Object(Symfony\Component\DependencyInjection\Definition), true) #8 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/AbstractRecursivePass.php(60): Symfony\Component\DependencyInjection\Compiler\ResolveParameterPlaceHoldersPass->processValue(Object(Symfony\Component\DependencyInjection\Definition), true) #9 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveParameterPlaceHoldersPass.php(79): Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass->processValue(Array, true) #10 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/AbstractRecursivePass.php(39): Symfony\Component\DependencyInjection\Compiler\ResolveParameterPlaceHoldersPass->processValue(Array, true) #11 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveParameterPlaceHoldersPass.php(43): Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass->process(Object(Symfony\Component\DependencyInjection\ContainerBuilder)) #12 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/Compiler.php(141): Symfony\Component\DependencyInjection\Compiler\ResolveParameterPlaceHoldersPass->process(Object(Symfony\Component\DependencyInjection\ContainerBuilder)) #13 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php(788): Symfony\Component\DependencyInjection\Compiler\Compiler->compile(Object(Symfony\Component\DependencyInjection\ContainerBuilder)) #14 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(637): Symfony\Component\DependencyInjection\ContainerBuilder->compile() #15 /home/hockeyla/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(135): Symfony\Component\HttpKernel\Kernel->initializeContainer() #16 /home/hockeyla/public_html/modules/autoupgrade/classes/UpgradeTools/SymfonyAdapter.php(74): Symfony\Component\HttpKernel\Kernel->boot() #17 /home/hockeyla/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader17.php(50): PrestaShop\Module\AutoUpgrade\UpgradeTools\SymfonyAdapter->initAppKernel() #18 /home/hockeyla/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader.php(85): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader17->initConstants() #19 /home/hockeyla/public_html/modules/autoupgrade/classes/TaskRunner/Upgrade/UpgradeDb.php(41): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader->doUpgrade() #20 /home/hockeyla/public_html/admin2015/autoupgrade/ajax-upgradetab.php(53): PrestaShop\Module\AutoUpgrade\TaskRunner\Upgrade\UpgradeDb->run() #21 {main}`

Any help is appreciated. Thanks!

**Comment by khouloudbelguith** [comment]
Hi @jlorquin,

This issue with the upgrade module, as reported in https://github.com/PrestaShop/autoupgrade/issues/298

Did you upgrade the module before starting an upgrade? The last version is v4.9.0

**Comment by jlorquin** [comment]
> Hi @jlorquin,
> 
> This issue with the upgrade module, as reported in [PrestaShop/autoupgrade#298](https://github.com/PrestaShop/autoupgrade/issues/298)
> 
> Did you upgrade the module before starting an upgrade? The last version is v4.9.0

Yes, it's the latest version. (I actually updated all modules before proceeding to PS update)

**Comment by khouloudbelguith** [comment]
@jlorquin, here's the PR: https://github.com/PrestaShop/PrestaShop/pull/14620 to fix the issue.
Your ticket is a duplicate of https://github.com/PrestaShop/PrestaShop/issues/14617#issuecomment-510884731
So, I close it.
Thanks!

**Comment by khouloudbelguith** [comment]
Duplicate of https://github.com/PrestaShop/PrestaShop/issues/14617#issuecomment-510884731

**Comment by jlorquin** [comment]
> @jlorquin, here's the PR: #14620 to fix the issue.
> Your ticket is a duplicate of [#14617 (comment)](https://github.com/PrestaShop/PrestaShop/issues/14617#issuecomment-510884731)
> So, I close it.
> Thanks!

So you're saying that I don't have autoupgrade v4.9.0 and that I have to unistall and install it it instead of  updating it?

**Comment by khouloudbelguith** [comment]
@jlorquin, your issue is the same reported in this ticket: https://github.com/PrestaShop/PrestaShop/issues/14617#issuecomment-510884731 with the same errors in the logs
We have an old PR: https://github.com/PrestaShop/autoupgrade/issues/298 
And this is the new PR: https://github.com/PrestaShop/PrestaShop/pull/14620 should be added to PS1.7.6.1.
Thanks!

**Comment by jlorquin** [comment]
> @jlorquin, your issue is the same reported in this ticket: [#14617 (comment)](https://github.com/PrestaShop/PrestaShop/issues/14617#issuecomment-510884731) with the same errors in the logs
> We have an old PR: [PrestaShop/autoupgrade#298](https://github.com/PrestaShop/autoupgrade/issues/298)
> And this is the new PR: #14620 should be added to PS1.7.6.1.
> Thanks!

Yes but I don't understand the solution well. The English used there is quite bad. One user is saying:

> Hi ,
> i resolved the problem
> the issue :
> when you look you’re module clik upgrade you see 4.9.0 but in fact you have the version 4.7.0
> 
> you must to restore you’re backup and you uninstall the module clik upgrade you downlode the module at prestashop you install it
> 
> you make the permission 777 after you upgrade and everithing it’s gone be ok !

Are you saying that is the solution? The permission 777 has to be applied to what folder exactly?

Thanks!
